### PR TITLE
Kangz/breakpad fixup

### PR DESCRIPTION
**Pull request by Kangz**
PTAL, small fixups to the breakpad code. Thanks!

### CODE REVIEW

**Comment by DolceTriade** [comment]
LGTM

File: daemon/src/engine/framework/CrashDump.cpp [hunk 1]
```diff
@@ -161,14 +159,10 @@ static bool CreateReportChannel(int& fdServer, int& fdClient) {
...
-bool BreakpadInit() {
-    if (!enableBreakpad.Get()) {
-        return false;
-    }
-
+static bool BreakpadInitInternal() {
     int fdServer, fdClient;
     if (!CreateReportChannel(fdServer, fdClient)) {
-        Log::Warn("Failed to start crash logging server");
+        crsahDumpLogs.Warn("Failed to start crash logging server");
```
Comment (slipher) [inline daemon/src/engine/framework/CrashDump.cpp]: Typo.

**Comment by slipher** [comment]
Looks good other than the typo.

File: daemon/src/engine/framework/CrashDump.cpp (hunk 1 above)
Comment (Kangz) [inline daemon/src/engine/framework/CrashDump.cpp]: Ok I'll fix.
Odd, I thought I tested compilation.

Comment (Kangz) [inline daemon/src/engine/framework/CrashDump.cpp]: It was in a if defined(__linux__) block. Done.

**Comment by Kangz** [comment]
Thanks for the review, merging.

**Pull request merged**